homeassistant/components/systemmonitor/sensor.py

```diff
@@ -361,11 +390,14 @@ async def async_setup_platform(
         sensor_registry[(type_, argument)] = SensorData(
             argument, None, None, None, None
         )
+        unique_id = sensor[CONF_UNIQUE_ID]
         entities.append(
-            SystemMonitorSensor(sensor_registry, SENSOR_TYPES[type_], argument)
+            SystemMonitorSensor(
+                sensor_registry, SENSOR_TYPES[type_], unique_id, argument
+            )
         )
 
-    scan_interval = config.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
+    scan_interval = DEFAULT_SCAN_INTERVAL
     await async_setup_sensor_registry_updates(hass, sensor_registry, scan_interval)
```

Comment:
```suggestion
    await async_setup_sensor_registry_updates(hass, sensor_registry, SCAN_INTERVAL)
```

I'd rename `DEFAULT_SCAN_INTERVAL` to `SCAN_INTERVAL` since its not a default anymore, it is the value

Reply:
I left it intentionally as in moving it to a Data Coordinator in next step but I guess you're right to rename for now.